content/influxdb/v2.0/reference/rc0-upgrade-guide.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Upgrade to InfluxDB OSS 2.0rc0
+description: >
+  Upgrade to InfluxDB OSS 2.0rc0.
+menu:
+  influxdb_2_0_ref:
+    name: Upgrade to InfluxDB OSS 2.0rc0
+weight: 9
+---
+
+To upgrade from InfluxDB 2.0 beta 16 or earlier to InfluxDB 2.0rc0, you must manually upgrade all resources and data to the latest version by completing the following steps:
+
+- [1. Disable existing integrations](#)
+- [2. Stop existing InfluxDB beta instance](#)
+- [3. (Optional) Rename existing InfluxDB binaries](#)
+- [4. Move existing data and start the latest InfluxDB](#)
+- [5. Start old InfluxDB beta instance](#)
+- [6. Create configuration profiles for the InfluxDB CLI](#)
+- [7. Copy all resources from old instance to the new one](#)
+- [8. Setup integrations to point to new instance](#)
+- [9. Load historical data into new instance](#)
+
+Depending on how you have things set up and how important the data stored in InfluxDB is,
+you can choose the parts of this guide that make sense to you.
+For example, if there's nothing in your existing InfluxDB beta instance you need to keep, feel free to skip the sections about migrating resources and data to the new instance.
+
+### Why is this manual process required?
+
+To ensure that existing InfluxQL integrations work with the latest release,
+we had to make breaking changes to the underlying storage engine for InfluxDB 2.0.
+
+{{% note %}}
+If you have questions upgrading please
```

Comment:
questions while upgrading? and add comma after upgrading. I also would cut "please". :)